src/nodedef.cpp

```diff
@@ -194,15 +216,26 @@ void TileDef::deSerialize(std::istream &is, u8 contentfeatures_version,
 	int version = readU8(is);
```

Comment:
`if (version < 6)` then unsupported. Otherwise this function will read garbage.

Reply:
Reasonable; the check should have been in `master` already.